scripts/nbconverted/main_Pa_sim_simple_nonlinear.py

```diff
@@ -0,0 +1,320 @@
+
+# coding: utf-8
```

Comment:
I am confused about how this is different than the linear simulation - also, a bunch of code is redundant between the two scripts

Reply:
Yes there is a lot of similarity between the "main" scripts because they are both running the same interpolation analysis but differ in the data they are using.  So the linear and nonlinear main scripts are supposed to be generating fake data by adding a very strong linear and nonlinear gene expression signal on top of the existing gene expression data from the compendium.  Ultimately we want to then compare the performance of VAE vs PCA vs Gene space in their ability to interpolate gene expression across samples.  Hopefully that makes some sense.